### PR TITLE
Adding bike article

### PR DESCRIPTION
**Pull request by atribolet**
Add Bike-Related GIS Data Resources article

### CODE REVIEW

File: _posts/2020-05-26-bike-related-gis-data-resources.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+Other bike-related information resources that are _coming soon_ (i.e., over the next few months):
+
+- **Wasatch Front Bike and Pedestrian Demand Score.** This is an update of the 2013 UCATS “Latent Demand” methodology that assigned a variety of demand-related indicators to the road network. [This dataset](http://wfrc.maps.arcgis.com/home/webmap/viewer.html?webmap=8ca837a9d3ff4b1299468e40b3eac383) will incorporate new and updated GIS data and will be published as a stand-alone layer that uses the road centerline geometry. Future enhancements building on this methodology are also planned by WFRC and partners.  
+- **Network Quality.** Metrics for nearby bike network quality have been implemented by UDOT and AGRC will also soon be available for the Wasatch Front metro area,
```
Comment (nathankota) [inline _posts/2020-05-26-bike-related-gis-data-resources.md:33]:
```suggestion
- **Network Quality.** Metrics for nearby bike network quality implemented by UDOT and AGRC will also soon be available for the Wasatch Front metro area, 
```

**Comment by steveoh** [comment]
@atribolet please move the images from `_posts` to `images`

**Comment by atribolet** [comment]
> @atribolet please move the images from `_posts` to `images`

Oh sorry about that! Didn't realize I'd added them there. Moving now. Thanks!

**Comment by atribolet** [comment]
K, this is ready for posting @steveoh

**Comment by atribolet** [comment]
@bertgranberg @BGranberg Hi, Bert! I wanted to make sure the placement of the images is ok for this article. Would you mind taking a look please? Thanks! 

Once Bert confirms, then this is good for finalizing @steveoh

**Pull request merged**